components/Route/RenderRoute.tsx

```diff
@@ -27,8 +27,10 @@ const RenderRoute = ({ route }: { route: Route }) => {
         id="pointLayer"
         filter={["==", "$type", "Point"]}
         style={{
-          circleColor: "black",
-          circleRadius: 5,
+          circleColor: "#eaeaea",
+          circleRadius: 6,
+          circleStrokeWidth: 2,
+          circleStrokeColor: "#000000",
```

Comment:
```suggestion
          circleStrokeColor: foofTheme["color-basic-500"],
```

Ich würde nicht schwarz nehmen, sondern eher ein mittleres grau